e2e_tests/conftest.py

```diff
@@ -65,17 +65,49 @@ def environment():
     )
 
 
+def initialize_container(container_type, project):
```

Comment:
maybe add `assert container_type in AVAILABLE_CONTAINERS` here, just to be safe and have a more explicit error? :)

or do the same as in `reinitialize_container`, i.e. `raise NotImplementedError` at the end of function body (preferably even `NotImplementedError(container_type)` to have more info in output)

Reply:
Sure

Reply:
Done